example/flask_op/views.py

```diff
@@ -224,12 +224,18 @@ def service_endpoint(endpoint):
     if request.method == 'GET':
         try:
             req_args = endpoint.parse_request(request.args.to_dict(), http_info=http_info)
-        except (InvalidClient, UnknownClient) as err:
```

Comment:
Are we sure that invalid and unknow client won't reiase here?

Reply:
That is the point, `InvalidClient` and `UnknownClient` are subclasses of `ClientAuthenticationError`. They will be handled by the new `except` statement.